manifests/dev/rbac.authorization.k8s.yaml.in

```diff
@@ -93,6 +93,7 @@ rules:
       - admissionregistration.k8s.io
     resources:
       - validatingwebhookconfigurations
+      - mutatingwebhookconfigurations
```

Comment:
release manifests need this as well.

Reply:
@davidvossel I wanted to ask you about it because I did not find any `validatingwebhookconfigurations` under release manifests, does its part of some general permissions? Also, I can see that release functional tests passed the stage of deployment.

Reply:
umm, that's not good. I'll take a look

Reply:
@cynepco3hahue  i figured out why. We have issues with our RBAC rules. #1415 will fix it. We'll need the mutation webhook privilege for virt-api in both once my PR gets merged